client/src/test/java/com/influxdb/client/ITBenchmarkTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * The MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.influxdb.client;
+
+import java.lang.management.ManagementFactory;
+import java.time.Duration;
+import java.time.Instant;
+
+import com.influxdb.client.benchmark.BenchmarkOptions;
+import com.influxdb.client.benchmark.ClientBenchmark;
+import com.influxdb.client.domain.Bucket;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+
+/**
+ * Client benchmark test
+ */
+@RunWith(JUnitPlatform.class)
+class ITBenchmarkTest extends AbstractITClientTest {
+
+    private BucketsApi bucketsApi;
+    private Bucket bucket;
+
+    @BeforeEach
+    void setUp() {
+
+        bucketsApi = influxDBClient.getBucketsApi();
+        OrganizationsApi organizationsApi = influxDBClient.getOrganizationsApi();
+
+        organizationsApi.findOrganizations()
+            .stream()
+            .filter(org -> org.getName().endsWith("-IT"))
+            .forEach(organizationsApi::deleteOrganization);
+
+        bucketsApi.findBuckets()
+            .stream()
+            .filter(bucket -> bucket.getName().endsWith("-IT"))
+            .forEach(bucket -> {
+                console("delete bucket: " + bucket.getName());
+                bucketsApi.deleteBucket(bucket);
+            });
+
+        bucket = bucketsApi.createBucket("benchmark-IT", findMyOrg());
```

Comment:
The bucket name has to be unique:

```suggestion
        bucket = bucketsApi.createBucket(generateName("benchmark"), findMyOrg());
```